torchio/data/image.py

```diff
@@ -377,10 +378,10 @@ def _parse_path(
             ) -> Optional[Union[Path, List[Path]]]:
         if path is None:
             return None
-        if isinstance(path, (str, Path)):
-            return self._parse_single_path(path)
-        else:
+        if isinstance(path, list):
```

Comment:
Did you mean `Iterable` here?

Reply:
Ops... I changed it at first to try something else and then forgot about it.
Can I use `git commit --amend` and then force push for this situation?

Reply:
You can just fix this in a new commit, don't worry about squashing or force-pushing!